node_cli/main.js

```diff
@@ -0,0 +1,80 @@
+const inquirer = require("inquirer");
+const tasks = require("./tasks");
+const { logout } = require("./realm/realmAuth");
+const index = require("./index");
+
+async function mainMenu(authedUser) {
+  inquirer
+    .prompt({
+      type: "rawlist",
+      name: "mainMenu",
+      message: "What would you like to do?",
+      choices: [
+        "Create a task",
+        "Show all of my tasks",
+        "Get a specific task",
+        "Change a task status",
+        "Edit a task",
+        "Delete a task",
+        "Watch for changes",
+        "Log out / Quit",
+      ],
+    })
+    .then(async (answers) => {
+      switch (answers.mainMenu) {
+        case "Create a task":
+          await tasks.createTask(authedUser).then(() => {
+            mainMenu(authedUser);
+          });
```

Comment:
Could clean these up, the `.then()` is redundant with the `await`

```js
case "Create a task":
  await tasks.createTask(authedUser);
  mainMenu(authedUser);
  break;
```

NOTE: You can also avoid the `break`s if you add curly brace scope to each case

```js
case "Create a task": {
  await tasks.createTask(authedUser);
  mainMenu(authedUser);
}
```

Reply:
Block scope does not preclude break!

Reply:
Oh oops Bush is definitely correct. I usually `return` values from switches which accomplishes the same thing as a `break`.